packages/standard_message_codec/example/lib/readme_excerpts.dart

```diff
@@ -16,5 +16,5 @@ void main() {
     'foo': true,
     3: 'fizz',
   });
-  print('The encoded message is $data');
+  print('The encoded message is $data'); // ignore: avoid_print
```

Comment:
This should be at the file level so this doesn't show up in the excerpt (once the excerpt annotations are fixed).